lib/libevent/libevent.nit

```diff
@@ -458,6 +477,38 @@ extern class ConnectionListener `{ struct evconnlistener * `}
 	end
 end
 
+# A listener acting on an Unix Socket, spawns `Connection` on new connections
+extern class UnixConnectionListener `{ struct evconnlistener * `}
+
```

Comment:
This may change depending on the convention for the module, but I personally find this extra blank line to be slightly confusing since it separates the class definition from its super.
If the module separates both, keep it that way, otherwise, I'd say remove this line

Reply:
OK will fix